### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by johansen** [vrruiz/wsse-js#2]
I'm interested in using this code, but no license is specified (other than declaring your copyright). Is it intended to be reused? If so, under what terms is reuse permitted?

**Comment by vrruiz** [comment]
Yes, it is intended to be reused. My parts are under the terms of an standard MIT license.

**Comment by vrruiz** [comment]
I'll add a LICENSE file.

**Comment by johansen** [comment]
Awesome!
